src/components/TodoList/TodoList.jsx

```diff
@@ -1,44 +1,120 @@
 import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+
 import './TodoList.scss';
 
-export const TodoList = () => (
-  <div className="TodoList">
-    <h2>Todos:</h2>
-
-    <div className="TodoList__list-container">
-      <ul className="TodoList__list">
-        <li className="TodoList__item TodoList__item--unchecked">
-          <label>
-            <input type="checkbox" readOnly />
-            <p>delectus aut autem</p>
-          </label>
-
-          <button
-            className="
-              TodoList__user-button
-              TodoList__user-button--selected
-              button
-            "
-            type="button"
-          >
-            User&nbsp;#1
-          </button>
-        </li>
-
-        <li className="TodoList__item TodoList__item--checked">
-          <label>
-            <input type="checkbox" checked readOnly />
-            <p>distinctio vitae autem nihil ut molestias quo</p>
-          </label>
-
-          <button
-            className="TodoList__user-button button"
-            type="button"
+export class TodoList extends React.Component {
+  state = {
+    todoTitle: '',
+    todoFilter: 'All',
+  }
+
+  handleTodos = (event) => {
+    const { name, value } = event.target;
+
+    this.setState({
+      [name]: value,
+    });
+  }
+
+  render() {
+    const { selectUser, todos } = this.props;
+    const { todoFilter, todoTitle } = this.state;
+    let filteredTodos;
+
+    switch (todoFilter) {
+      case 'Active':
+        filteredTodos = todos.filter(todo => !todo.completed);
+        break;
+
+      case 'Completed':
+        filteredTodos = todos.filter(todo => todo.completed);
+        break;
+
+      default:
+        filteredTodos = todos;
+        break;
+    }
+
+    if (todoTitle.length > 0) {
+      filteredTodos = filteredTodos.filter((todo) => {
```

Comment:
For the future - it's better to make all operations (filtering) in the upper level, and pass ALREADy prepared todos to 'TodoList' component. It should only render the list